java/client/src/main/java/glide/api/commands/ServerManagementCommands.java

```diff
@@ -133,4 +133,18 @@ public interface ServerManagementCommands {
      * }</pre>
      */
     CompletableFuture<String[]> time();
+
+    /**
+     * Returns <code>unix time</code> of the last DB save timestamp or startup timestamp if no save
```

Comment:
The review checklist mentions the following, is it applicable here or no?
`For standalone keyless commands: Document whether the command is distributed to all nodes or only to the primary (e.g., ACL commands).`

Reply:
Cluster client implements `ServerManagementClusterCommands` interface, but this one goes for standalone client - routing is not applicable to it.
Please, review docs for cluster client.

Reply:
Yeah I thought routing wasn't applicable to standalone. In that case, why does the review checklist specify that standalone keyless commands should document this?

Reply:
I mistake I suppose. Routing is applicable for cluster client (and cluster interfaces) only.